packages/execution-context/src/lib/contextStore.ts

```
@@ -0,0 +1,75 @@
+   */
+  public static async withContext<T = void>(
+    context: ExecutionContext,
+    function_: () => Promise<T>,
+  ): Promise<T> {
+    return await new Promise((resolve, reject) => {
+      ContextStore.getStorage().run(context, () => {
+        function_().then(resolve).catch(reject);
+      });
+    });
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)